src/cms/widget-create.md

```diff
@@ -2,79 +2,90 @@
 
-    - Set **Assign to Store Views** to `All Store Views`, or to the view where the app will be available. To select more than one, hold the Ctrl key down and select each option.
+   - For **Assign to Store Views**, select the store views where the widget will be visible.
 
-    - Enter a number in the **Sort Order** field to determine the order of the block if it appears in the same location as other content elements. The top position is zero.
+      You can select a specific store view, or `All Store Views`. To select multiple views, hold down the Ctrl key (PC) or the Command key (Mac) and click each option.
 
-    ![]({% link images/images/widget-storefront-properties.png %}){: .zoom}
-    _Storefront Properties_
+   - (Optional) For **Sort Order**, enter a number to determine the order this item appears with others in the same part of the page. (`0` = first, `1` = second, `3` = third, and so on.)
 
-## Step 2: Specify Where It Goes
+   ![]({% link images/images/widget-storefront-properties.png %}){: .zoom}
+   _Storefront properties_
 
-1. In the Layout Updates section, click <span class="btn">Add Layout Update</span>.
+1. In the _Layout Updates_ section, click <span class="btn">Add Layout Update</span>.
 
 1. Set **Display On** to the type of page where it is to appear.
 
-1. In the **Block Reference** list, choose the area of the page layout where it is to be placed.
+1. In the **Container** list, choose the area of the page layout where it is to be placed.
 
-    ![]({% link images/images/widget-layout-update-home-page.png %}){: .zoom}
-    _Layout Updates_
+   ![]({% link images/images/widget-layout-update-home-page.png %}){: .zoom}
+   _Layout updates_
 
-1. If the widget is a link, set <b>Template </b>to one of the following:
+1. If the widget is a link, set **Template** to one of the following:
 
-    | Block Template | Formats the content so it can be placed as standalone unit on the page. |
-    | Inline Template | Formats the content so it can be placed inside other content. For example, a link that goes inside a paragraph of text. |
-    {:style="table-layout:auto"}
+   | Block Template | Formats the content so it can be placed as standalone unit on the page. |
+   | Inline Template | Formats the content so it can be placed inside other content. For example, a link that goes inside a paragraph of text. |
 
-## Step 3: Complete the Options
+## Step 3: Complete the widget options
 
 The options for each widget type vary slightly, but the process is essentially the same. The following example displays the product list for a specific category, with pagination controls.
 
-1. In the panel on the left, choose **Widget Options**.
+1. In the left panel, choose **Widget Options**.
 
 1. Click <span class="btn">Select Block</span>.
 
-1. Enter a **Title** to appear above the list. For example, `Featured Products`.
+1. Enter a **Title** to appear above the list.
+
+   For example, `Featured Products`.
```

Comment:
this isn't a sentence.    suggest moving to first line with , such as or e.g.

Reply:
good call